pkg/network/tracer/tracer.go

```diff
@@ -182,6 +185,13 @@ func newTracer(cfg *config.Config, telemetryComponent telemetryComponent.Compone
 	tr.reverseDNS = newReverseDNS(cfg, telemetryComponent)
 	tr.usmMonitor = newUSMMonitor(cfg, tr.ebpfTracer)
 
+	connectionProtocolMap := tr.ebpfTracer.GetMap(probes.ConnectionProtocolMap)
```

Comment:
Should this only be enabled if protocol classification is enabled?

Reply:
yes sorry I put this back in draft, it needs a bit more work